clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBServerConfig.java

```diff
@@ -375,6 +378,8 @@ public RocksDBServerConfig(VeniceProperties props) {
     this.atomicFlushEnabled = props.getBoolean(ROCKSDB_ATOMIC_FLUSH_ENABLED, true);
     this.separateRMDCacheEnabled = props.getBoolean(ROCKSDB_SEPARATE_RMD_CACHE_ENABLED, false);
 
+    this.blobTransferBatchOnlyEnabled = props.getBoolean(ENABLE_BLOB_TRANSFER_FOR_BATCH_ONLY_STORE, false);
```

Comment:
This config doesn't seem to be right. This `RocksDBServerConfig` is for host-level configs not store-level. Would `StoragePartitionConfig` or `VeniceStoreVersionConfig` work?

Reply:
I didn't find the docs clear on what the configs were used for. I used `RocksDBServerConfig` because the [RocksDBSstFileWriter](https://github.com/linkedin/venice/blob/ca88d722de394beea99371486e294025574546a1/clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBSstFileWriter.java#L121) already had access to i, and we wouldn't have to make so many changes. I can refactor to use `StoragePartitionConfig` or `VeniceStoreVersionConfig`

Reply:
Please consider using latter two since they are more close to `store-level` config. Though `RocksDBServerConfig` is easy to access w/o touching many codes, this is just not the right config and can impact code readability/maintainability